source/getting-started.rst

```diff
@@ -96,7 +96,7 @@ Using Selenium to write tests
 
 Selenium is mostly used for writing test cases.  The `selenium` package itself
 doesn't provide a testing tool/framework.  You can write test cases using
-Python's unittest module.  The other options for a tool/framework are py.test
+Python's unittest module.  The other options for a tool/framework are pytest
 and nose.
```

Comment:
Can you please add links to pytest & nose websites here?
- https://docs.pytest.org/en/stable/
- https://nose.readthedocs.io/en/latest/

Reply:
Sure

Reply:
```suggestion
Python's unittest module.  The other options for a tool/framework are `pytest <https://docs.pytest.org/en/stable/>`_
and `nose <https://nose.readthedocs.io/en/latest/>`_.
```